src/components/dropdowns/dropdowns.html

```diff
@@ -0,0 +1,61 @@
+<section class="docs-block">
+    <aside class="docs-block__info">
+        <h3 class="docs-block__header">Dropdowns</h3>
+    </aside>
+    <div class="docs-block__content">
+        <div class="mint-dropdown">
+            <div class="mint-dropdown__label">All options</div>
```

Comment:
@mr-mig i just checked it, there is different font-family for mint-label and for label in dropdown (here is the same as in selects), i am not sure anymore if i should use `mint-label` here